.changeset/miniflare-vitest-migration.md

```diff
@@ -0,0 +1,5 @@
+---
+"miniflare": patch
```

Comment:
This PR does not need a changeset as there is nothing to release. Please remove this file.